package.json

```
@@ -65,9 +65,9 @@
     "typescript": "^4.0.0"
   },
   "peerDependencies": {
-    "antd": ">= 5.8.2",
-    "react": ">=16.9.0",
-    "react-dom": ">=16.9.0"
+    "antd": ">=6.0.0",
```

```suggestion
    "antd": ">=6.1.0",
```

6.0.0 的antd 那边仍然冗余了包